github/ipld.yml

```diff
@@ -3177,43 +3123,17 @@ teams:
       #  - be familiar with GitHub Management
       #  - be ready to triage/review org configuration change request in github-mgmt
       maintainer:
-        - aschmahmann
         - rvagg
-      member:
-        - BigLep
-        - lidel
-        - willscott
```

Comment:
Simplify this down to members only?

Folks we want:
aschmahmann # IP Shipyard
rvagg # PL Inc
vmx # PL Inc
willscott # PL Inc